### DISCUSSION TITLE
Why can't remote Apple computers hear sound?

### DISCUSSION THREAD

**Discussion by ChicoSir** [rustdesk/rustdesk#6069] [Q&A]
RustDesk still cannot hear the sound played remotely when the mac is connected to the mac, but when the mac is connected to Windows, the Windows can play the sound normally without any problem. The current version of RustDesk is 1.2.3 and the mac version is 13.6.

I would like to ask if there are other solutions at present.
Because a lot of videos are going to play and the video needs to be able to play remote sound.
If it can be resolved, thank you very much.

todesk can play sound when connected properly

![v2-0d60114151e1453c89aee1de5b80b381_r](https://github.com/rustdesk/rustdesk/assets/44128292/f05b12f0-20e0-42c7-8574-f8742b6b0064)
_Muted off🔇_

![v2-171cbcbb0f8f0d06c408c20f307811ba_r](https://github.com/rustdesk/rustdesk/assets/44128292/91e35a11-2ad6-4cb3-b3d9-2a818868c234)
_Playing music on the other party's computer and still unable to hear it, I have tried to switch the sound output interface, and the same applies_

**Comment by discoveryX64** [comment]
My only guess (and I have only 1 mac machine so I can't test this exactly how you did this,) is you have a audio software or something open. When I have my headphone software open on my pc, it doesn't project audio to my mac, but when I exit the headphone software, it works.
